src/app.ts

```diff
@@ -35,8 +37,9 @@ app.use(compress());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(favicon(path.join(app.get('public'), 'favicon.ico')));
-// Host the public folder
-app.use('/', express.static(app.get('public')));
```

Comment:
is there a difference between `app.get('public')` and just `'public'`? Since you moved this to the routes folder just wanted to make sure the behavior is the same

Reply:
I also think that it is the same but that is from @eddiejaoude so I left it there.
Maybe he can explain.